daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -17,43 +17,47 @@
 
 package org.apache.daffodil.japi
 
-import org.apache.daffodil.compiler.{ Compiler => SCompiler }
+import org.apache.daffodil.compiler.{Compiler => SCompiler}
 import org.apache.daffodil.japi.debugger._
 import org.apache.daffodil.japi.logger._
 import org.apache.daffodil.japi.packageprivate._
 import org.apache.daffodil.japi.infoset._
 import org.apache.daffodil.japi.io.InputSourceDataInputStream
-import org.apache.daffodil.debugger.{ InteractiveDebugger => SInteractiveDebugger }
-import org.apache.daffodil.debugger.{ TraceDebuggerRunner => STraceDebuggerRunner }
-import org.apache.daffodil.api.{ Diagnostic => SDiagnostic }
+import org.apache.daffodil.debugger.{InteractiveDebugger => SInteractiveDebugger}
+import org.apache.daffodil.debugger.{TraceDebuggerRunner => STraceDebuggerRunner}
+import org.apache.daffodil.api.{Diagnostic => SDiagnostic}
+
 import scala.collection.JavaConverters._
 import java.io.File
 import java.nio.channels.Channels
 import java.nio.channels.ReadableByteChannel
 import java.nio.channels.WritableByteChannel
+
 import scala.collection.JavaConversions.seqAsJavaList
-import org.apache.daffodil.api.{ DataLocation => SDataLocation }
-import org.apache.daffodil.api.{ Diagnostic => SDiagnostic }
-import org.apache.daffodil.api.{ LocationInSchemaFile => SLocationInSchemaFile }
-import org.apache.daffodil.api.{ WithDiagnostics => SWithDiagnostics }
-import org.apache.daffodil.compiler.{ ProcessorFactory => SProcessorFactory }
-import org.apache.daffodil.processors.{ DataProcessor => SDataProcessor }
-import org.apache.daffodil.processors.{ ParseResult => SParseResult }
-import org.apache.daffodil.processors.{ UnparseResult => SUnparseResult }
-import org.apache.daffodil.util.{ ConsoleWriter => SConsoleWriter }
-import org.apache.daffodil.util.{ FileWriter => SFileWriter }
-import org.apache.daffodil.util.{ LogWriter => SLogWriter }
-import org.apache.daffodil.util.{ LoggingDefaults => SLoggingDefaults }
-import org.apache.daffodil.util.{ NullLogWriter => SNullLogWriter }
-import org.apache.daffodil.externalvars.ExternalVariablesLoader
+import org.apache.daffodil.api.{DataLocation => SDataLocation}
+import org.apache.daffodil.api.{Diagnostic => SDiagnostic}
```

Comment:
Looks like yoru auto-import formatter broke? It changed everything to not use spaces?

Reply:
Fixed. There is a setting for this. It's not set by default. Added a comment to the IDEA page on wiki.